src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -151,7 +154,9 @@ private void gitCommitAndPush(Params params, String commitMessage) {
         buildToolStrategy.includeWrapperFiles(changes);
         changes.forEach(c -> execGitCmd(execOperations, params.gitCheckoutDir, "add", c.toPath().toString()));
         execGitCmd(execOperations, params.gitCheckoutDir, "checkout", "--quiet", "-b", params.prBranch);
-        execGitCmd(execOperations, params.gitCheckoutDir, "commit", "--quiet", "-s", "-m", commitMessage);
+        var argsAndExtra = new ArrayList<>(List.of("commit", "--quiet", "-s", "-m", commitMessage));
```

Comment:
`-s` should not be part of the default options. But we set it explicitly in our Solutions builds via the new extra args.